src/pybind/mgr/nfs/ganesha_conf.py

```diff
@@ -179,8 +179,9 @@ def parse(self) -> List[RawBlock]:
 
 
 class FSAL(object):
-    def __init__(self, name: str) -> None:
+    def __init__(self, name: str, cmount_path: Optional[str] = "/") -> None:
```

Comment:
I think we should add a comment here stating the reason why the default is kept as `/`

Reply:
> I think we should add a comment here stating the reason why the default is kept as `/`

Added a code comment!